src/main/java/seedu/address/ui/WidgetViewBox.java

```diff
@@ -25,44 +28,72 @@
     @FXML
     private Label textTwo;
     @FXML
-    private Label divThree;
-    @FXML
     private Label textThree;
     @FXML
-    private Label divFour;
-    @FXML
-    private Label textFour;
-    @FXML
-    private Label divFive;
-    @FXML
-    private Label textFive;
-    @FXML
-    private Label divSix;
+    private Label divThree;
+    //TODO: This is a stop gap to display everything about the client's notes.
     @FXML
-    private Label textSix;
+    private TextArea textFour;
     @FXML
     private Label footer;
+    private WidgetObject widgetObject;
+    private TextClock textClock;
 
     /**
      * Creates a {@code WidgetViewBox} with the given {@code WidgetObject}.
      */
-    public WidgetViewBox(WidgetObject widgetObject) {
+    public WidgetViewBox() {
         super(FXML);
-        this.widgetObject = widgetObject;
-        header.setText(widgetObject.header());
-        divOne.setText(widgetObject.divOne());
-        divTwo.setText(widgetObject.divTwo());
-        divThree.setText(widgetObject.divThree());
-        divFour.setText(widgetObject.divFour());
-        divFive.setText(widgetObject.divFive());
-        divSix.setText(widgetObject.divSix());
-        textOne.setText(widgetObject.textOne());
-        textTwo.setText(widgetObject.textTwo());
-        textThree.setText(widgetObject.textThree());
-        textFour.setText(widgetObject.textFour());
-        textFive.setText(widgetObject.textFive());
-        textSix.setText(widgetObject.textSix());
-        footer.setText(widgetObject.footer());
+        textClock = new TextClock(header);
+        textClock.play();
+        divOne.setText(Locale.getDefault().getDisplayCountry());
+        textOne.setText(TimeZone.getDefault().getDisplayName());
+        divTwo.setText("Travelling BusinessMan");
+        textTwo.setText("");
+        textThree.setText("");
+        divThree.setText("");
+        textFour.setText("");
+        footer.setText("Made in NUS");
```

Comment:
Footer should probably be lowered to the bottom border right, this isn't that impt for now since it's functional! 


![image](https://user-images.githubusercontent.com/38996397/96711510-4237a380-13d0-11eb-9de0-27bcd8749e54.png)